### DISCUSSION TITLE
Soft Launch 8.0.0?

### DISCUSSION THREAD

**Discussion by faanskit** [icanos/hassio-plejd#195] [Ideas]
Hey,

With all breaking changes coming in 8.0.0. Should it maybe be "released" as 8.0.0-PUBLIC-BETA to signal that this is something special.

/Marcus

**Comment by TorulfDanielsson** [comment]
For the record. Updated to 0.8.0-beta on core-2021.6.4/supervisor-2021.06.0/Home Assistant OS 6.0 installed on Intel NUC DC3217IYE I3 3217U - intel WLAN/BT 6235.
Reconfigured Entity-id as described and all seems to be up and running without issues.
Did not know where to give feedback - pleas point me in the right direction if this is the wrong place.

**Comment by SweVictor** [comment]
Perfect, thanks for that!
